plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts

```diff
@@ -112,6 +112,13 @@ export class TaskWorker {
   }
 
   start() {
+    const recoverTasksId = setInterval(async () => {
```

Comment:
I think this might prove problematic, and you might get weird behaviour, such as overlapping calls, and it would be good to handle exceptions.  I think having a promise that wraps `setTimeout` might be better.